stwcs/wcsutil/headerlet.py

```diff
@@ -1284,11 +1287,12 @@ def delete_headerlet(filename, hdrname=None, hdrext=None, distname=None,
     for f in filename:
         print("Deleting Headerlet from ", f)
         _delete_single_headerlet(f, hdrname=hdrname, hdrext=hdrext,
-                                 distname=distname, logging=logging, logmode='a')
+                                 distname=distname, delete_all=delete_all,
+                                 logging=logging, logmode='a')
 
 
 def _delete_single_headerlet(filename, hdrname=None, hdrext=None, distname=None,
-                             logging=False, logmode='w'):
+                             delete_all=False, logging=False, logmode='w'):
```

Comment:
I agree that this internal (Python-private) function is not correctly named since it will actually delete multiple extensions if found.  This will be addressed in the next commit.

Reply:
Fixed in 9098e78